tests/test_e2e_40_sdntrace.py

```diff
@@ -942,3 +942,51 @@ def test_080_validate_attribute_on_payload(self):
         api_url = KYTOS_API + '/amlight/sdntrace_cp/v1/traces'
         response = requests.put(api_url, json=payload)
         assert response.status_code == 200
+
+    def test_085_test_evcs_terminating_on_nnis(cls):
+        "Test EVCs terminating on NNIs"
+
+        cid1 = cls.create_evc(999, "00:00:00:00:00:00:00:02:1", "00:00:00:00:00:00:00:04:1")
```

Comment:
@gretelliz, on second thought, if we'll keep this to represent the same circuit that Italo also used in the issue, then maybe it should be traced too? otherwise, it can lead to confusion down to road for whoever will read/maintain/extend this test case.